resteasy-core/src/main/java/org/jboss/resteasy/plugins/providers/MultiValuedCollectionParamConverter.java

```diff
@@ -0,0 +1,139 @@
+package org.jboss.resteasy.plugins.providers;
+
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Method;
+import java.util.Collection;
+
+import javax.annotation.Priority;
+import javax.ws.rs.ext.ParamConverter;
+import javax.ws.rs.ext.Provider;
+
+import org.jboss.resteasy.core.StringParameterInjector;
+import org.jboss.resteasy.resteasy_jaxrs.i18n.Messages;
+
+/**
+ * @author Marek Kopecky mkopecky@redhat.com
+ * @author Ron Sigal rsigal@redhat.com
+ */
+@Provider
+@Priority(Integer.MAX_VALUE)
+public class MultiValuedCollectionParamConverter implements ParamConverter<Collection<?>> 
+{
+   private Constructor<?> constructor;
+   private StringParameterInjector stringParameterInjector;
+   private String separator;
+
+   public MultiValuedCollectionParamConverter(Class<?> rawType, Annotation[] annotations, Constructor<?> constructor, StringParameterInjector stringParameterInjector, String separator)
```

Comment:
Quick clean up for both `rawType` and `annotations` parameters that are not used here

Reply:
Done.